### ISSUE TITLE
Add multiple databases to search list.

### ISSUE THREAD

**Issue by chhajedji** [dkogan/xcscope.el#32]
I have multiple cscope databases to refer to at once. Is there any way in which I can add all the database to the search list? Basically, I want to add multiple directories to `cscope-set-initial-directory`, but once I add a new directory, only that directory remains, and the previous ones are discarded. Same functionality can be achieved in vim by the command `:cscope add <path to database>`

Please excuse if this is a rookie question as I'm not very familiar with lisp.

**Comment by dkogan** [comment]
Hi. Sorry it took so long to reply; I've been busy. So you want more
than one cscope database to be used at once? I.e. we would search one
database, and then search the other, and the other, and so on? If that's
what you want, xcscope doesn't currently support it. It wouldn't be TOO
hard to add that, but I don't have the time to do that at all.

If you want something else, please clarify.

**Comment by chhajedji** [comment]
Yes, that's precisely what I was looking for. As an alternative, I've added
all the database files in one cscope.files and created and added that
single cscope.out file. It would have been really nice to have the feature
to add multiple databases as Vim's cscope plugin allow this functionality.
